RELEASE.md

```diff
@@ -32,8 +32,8 @@ Maintaining the release branches for older minor releases happens on a best effo
   * kube-state-metrics image tag used in Kubernetes deployment yaml config.
 * Cut the new release branch, i.e., `release-1.2`, or merge/cherry-pick changes onto the minor release branch you intend to tag the release on
 * Cut the new release tag, i.e., `v1.2.0-rc.0`
-* Create issue requesting for image release to `k8s.gcr.io` (or to `staging-k8s.gcr.io` in case of release candidates) and assign it to one of Googlers (@serathius, @loburm)
-* Build and push newest image to `quay.io`(@brancz)
+* New images are automatically built and pushed to `gcr.io/k8s-staging-kube-state-metrics/kube-state-metrics`
+* Build and push newest image to `quay.io/coreos`(@brancz, @lilic)
```

Comment:
I think it would be better to use image build by CI instead of having owners build it locally. We could add script to sync image from k8s.gcr.io/kube-state-metrics to quay.io/coreos

Reply:
Yes good idea, will open issue for this!